include/exadg/operators/operator_base.cpp

```diff
@@ -849,6 +866,19 @@ OperatorBase<dim, Number, n_components>::reinit_cell(IntegratorCell &   integrat
                                                      unsigned int const cell) const
 {
   integrator.reinit(cell);
+
+  reinit_cell_additional(integrator, cell);
```

Comment:
Is `additional` a good name? I would intuitively choose `auxiliary`, but maybe there are even more precise words?

Reply:
The wording should mean "in addition to what is reinitialized in the base class". I am not sure if "auxiliary" can reflect this? An alternative I thought about was "derived", in the sense that this is reinitialization done by "derived" classes.

Reply:
Yes, I like "derived".

Reply:
I consider both options equivalent, so I will change this. Overall, these are all private member functions, so I think it is not super-important right now that we fix the naming.